contracts/arbitration/GTCR.sol

```diff
@@ -0,0 +1,1013 @@
+/* solium-disable max-len*/
+
+pragma solidity ^0.5.10;
+
+import "./Arbitrable.sol";
```

Comment:
We can remove these contracts and import from the `@kleros/erc-792` package in the form:
```
import { IArbitrable, Arbitrator } from "@kleros/erc-792/contracts/Arbitrator.sol"
```
This will require manually adding some variables like `arbitrator` and `arbitratorExtraData`